### DISCUSSION TITLE
活动图功能无法使用

### DISCUSSION THREAD

**Discussion by xinyuye66** [LmeSzinc/AzurLaneAutoScript#1336] [Q&A] [answered]
想问大佬 主线图功能正常运作
活动图功能打完第一个敌人后
为何会出现卡死在获得道具页面
![image](https://user-images.githubusercontent.com/109322219/179127155-13b39b0b-7434-4eec-83ec-b501837203ba.png)
![image](https://user-images.githubusercontent.com/109322219/179127190-83f466c6-5517-41b7-87d4-6d606f4680c2.png)

**Comment by NightSparrows** [comment]
我把module/combat/combat.py裡的drop=drop刪掉就可以了
但應該不是作者的用途
![image](https://user-images.githubusercontent.com/109325365/179136413-3a0e8192-4d9d-4f05-89ae-8c80c49f7b5e.png)
等大佬解決吧

**Comment by LmeSzinc** [comment, marked as answer]
https://github.com/LmeSzinc/AzurLaneAutoScript/commit/76a7eb4d084653b6a59f6a4c60d36fa290df2dde

已经修了，忘说了
